OPAL/ll/src/main/scala/org/opalj/ll/llvm/value/BasicBlock.scala

```diff
@@ -0,0 +1,69 @@
+/* BSD 2-Clause License - see OPAL/LICENSE for details. */
+package org.opalj.ll.llvm.value
+
+import org.bytedeco.llvm.LLVM.{LLVMBasicBlockRef, LLVMValueRef}
+import org.bytedeco.llvm.global.LLVM._
+import org.opalj.graphs.Node
+
+case class BasicBlock(block_ref: LLVMBasicBlockRef)
```

Comment:
Should we have a language-independent base class for basic blocks and similar common concepts?

Reply:
I guess so, but I wonder whether that would be out of scope for this PR.